Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "opentelemetry-swift",
+    platforms: [
+        .macOS(.v10_13),
+        .iOS(.v11),
+        .tvOS(.v11)
```

Comment:
question/ Don't we need also visionOS and macOS? Otherwise `dd-sdk-ios/Package.swift` won't build, no?

suggestion/ If we do `v12` for iOS and tvOS won't it solve problems with Xcode 15 build?

Reply:
this makes sense, we don't have to deal with version issues, I just pushed a commit on this.

Also updated Swift version.